server/server/session/src/main/java/com/alipay/sofa/registry/server/session/providedata/PushEfficiencyImproveService.java

```diff
@@ -0,0 +1,105 @@
+ */
+package com.alipay.sofa.registry.server.session.providedata;
+
+import com.alipay.sofa.registry.common.model.constants.ValueConstants;
+import com.alipay.sofa.registry.common.model.metaserver.ProvideData;
+import com.alipay.sofa.registry.log.Logger;
+import com.alipay.sofa.registry.log.LoggerFactory;
+import com.alipay.sofa.registry.server.session.bootstrap.SessionServerConfig;
+import com.alipay.sofa.registry.server.session.push.ChangeProcessor;
+import com.alipay.sofa.registry.server.session.push.FirePushService;
+import com.alipay.sofa.registry.server.session.push.PushEfficiencyImproveConfig;
+import com.alipay.sofa.registry.server.session.push.PushProcessor;
+import com.alipay.sofa.registry.server.shared.providedata.AbstractFetchSystemPropertyService;
+import com.alipay.sofa.registry.server.shared.providedata.SystemDataStorage;
+import com.alipay.sofa.registry.util.JsonUtils;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * @author jiangcun.hlc@antfin.com
+ * @since 2023/2/21
+ */
+public class PushEfficiencyImproveService
+    extends AbstractFetchSystemPropertyService<PushEfficiencyImproveService.SwitchStorage> {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(PushEfficiencyImproveService.class);
+  @Autowired private SessionServerConfig sessionServerConfig;
+
+  @Autowired ChangeProcessor changeProcessor;
+  @Autowired PushProcessor pushProcessor;
+  @Autowired FirePushService firePushService;
+
+  public PushEfficiencyImproveService() {
+    super(
+        ValueConstants.CHANGE_PUSH_TASK_DELAY_CONFIG_DATA_ID,
+        new SwitchStorage(INIT_VERSION, new PushEfficiencyImproveConfig()));
+  }
+
+  @Override
+  protected int getSystemPropertyIntervalMillis() {
+    return sessionServerConfig.getSystemPropertyIntervalMillis();
+  }
+
+  @Override
+  protected boolean doProcess(SwitchStorage expect, ProvideData data) {
+    final String configString = ProvideData.toString(data);
+
+    PushEfficiencyImproveConfig pushEfficiencyImproveConfig = new PushEfficiencyImproveConfig();
+    if (StringUtils.isNotBlank(configString)) {
+      try {
+        pushEfficiencyImproveConfig =
+            JsonUtils.read(configString, PushEfficiencyImproveConfig.class);
+      } catch (Throwable e) {
+        LOGGER.error("Decode PushEfficiencyImproveConfig failed", e);
+        return false;
+      }
+    }
+    if (pushEfficiencyImproveConfig == null || !pushEfficiencyImproveConfig.validate()) {
+      LOGGER.error(
+          "Fetch PushEfficiencyImproveConfig invalid, value={}", pushEfficiencyImproveConfig);
+      return false;
+    }
+
+    pushEfficiencyImproveConfig.setSessionServerConfig(sessionServerConfig);
+
+    SwitchStorage update = new SwitchStorage(data.getVersion(), pushEfficiencyImproveConfig);
+    if (!compareAndSet(expect, update)) {
+      return false;
+    }
+    changeProcessor.setWorkDelayTime(pushEfficiencyImproveConfig);
```

Comment:
i think we should use changeProcessor to fetch pushEfficiencyImproveConfig, PushEfficiencyImproveService dependency on  changeProcessor may not reasonable.